src/main/java/frc/robot/subsystems/manipulator/ManipulatorSubsystem.java

```diff
@@ -204,6 +204,10 @@ public Command getPodiumShootCommand() {
     return getShootCommand(ArmPosition.PODIUM_SHOOT, ShooterVelocity.PODIUM_SHOOT);
   }
 
+  public Command getLineShootCommand() {
+    return getShootCommand(ArmPosition.LINE_SHOOT, ShooterVelocity.SUBWOOFER_SHOOT);
```

Comment:
Should we make a ``ShooterVelocity.LINE_SHOOT``? Even if It is the same velocity.